app/_how-tos/generate-certificate-with-sni.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Associate a Certificate with an SNI
+content_type: how_to
+related_resources:
+  - text: SNI entity
+    url: /gateway/entities/snis/
+  - text: Certificates
+    url: /gateway/entities/certificate/
```

Comment:
```suggestion
  - text: Secure your API with TLS using SNIs
    url: app/_how-tos/secure-api-with-tls/
```
The `entities` key adds links to all entities in the info box, so entity links are redundant hre.